requirements_for_test.txt

```diff
@@ -20,5 +20,6 @@ black==21.5b2
 locust==2.10.1
 mypy==0.812
 sqlalchemy-stubs==0.4
```

Comment:
Should we still have the old stubs package here?  From the sqlalchemy2-stubs README:

> The stubs replace the use of the "sqlalchemy-stubs" package published by Dropbox.

It also mentions a mypy extension to go with this - do we need that as well?

Reply:
> It also mentions a mypy extension to go with this - do we need that as well?
We've got it already here: https://github.com/cds-snc/notification-api/blob/main/requirements_for_test.txt#L21
And we are running it in our CI here: https://github.com/cds-snc/notification-api/blob/main/scripts/run_tests.sh#L35

Reply:
> Should we still have the old stubs package here?

Interesting that they say they replace the old stubs. When I removed the old ones I was getting errors. But I'll try again and see what happens.